packages/evm/src/eof/verify.ts

```diff
@@ -101,31 +101,21 @@ function validateOpcodes(
   opcodeNumbers.add(0xfe)
 
   // Remove CODESIZE, CODECOPY, EXTCODESIZE, EXTCODECOPY, EXTCODEHASH, GAS
-  opcodeNumbers.delete(0x38)
-  opcodeNumbers.delete(0x39)
-  opcodeNumbers.delete(0x5a)
-  opcodeNumbers.delete(0x3b)
-  opcodeNumbers.delete(0x3c)
-  opcodeNumbers.delete(0x3f)
-
+  const a1 = [0x38, 0x39, 0x5a, 0x3b, 0x3c, 0x3f]
```

Comment:
What is all of this doing?  Are we just saving Lines of Code/character counts to make this change?  

Bigger picture, is there a more comprehensible way to organize our lists of valid opcodes per EVM context (i.e. what opcodes are valid under EOF only and what opcodes are valid in legacy EVM context)?  This whole code section is kind of hard to understand as to why we are deleting certain opcodes from the valid list.  A "better way" in my opinion would be to have two configurations, one valid for EOF and one for legacy EVM mode.

Reply:
Have removed the commit